src/sasjs-servicepack/deploy.js

```diff
@@ -0,0 +1,122 @@
+import path from 'path'
+import SASjs from '@sasjs/adapter/node'
+import chalk from 'chalk'
+import {
+  readFile,
+  folderExists,
+  createFile,
+  createFolder
+} from '../utils/file-utils'
+import { displayResult } from '../utils/displayResult'
+import {
+  getAccessToken,
+  findTargetInConfiguration
+} from '../utils/config-utils'
+
+export async function servicePackDeploy(
+  jsonFilePath = null,
+  targetName = null,
+  isForced = false
+) {
+  console.log({
+    jsonFilePath,
+    targetName,
+    isForced
+  })
+
+  if (jsonFilePath.split('.').slice(-1)[0] !== 'json') {
```

Comment:
You can use path.extname() to do this in a cross-platform way.


https://nodejs.org/api/path.html#path_path_extname_path